.profileType = profileType

```diff
@@ -0,0 +1,236 @@
+
```

Comment:
Context on what this file is?
Or, if it wasn't meant to be submitted, git rm + git ignore.

Ditto for the file named " er(String profileType, long id, String name, String school, String major) {". Looks like you might have pasted some code into the command line with a ">" :P